tests/unit/test-util/test.updates.js

```diff
@@ -0,0 +1,30 @@
+/* @flow */
+import {it, describe} from 'mocha';
+import {assert} from 'chai';
+import sinon from 'sinon';
+
+import {checkForUpdates} from '../../../src/util/updates';
+
+describe('util/updates', () => {
+  describe('checkForUpdates()', () => {
+    it('calls the notifier with the correct parameters', () => {
+      let updateNotifierStub = sinon.spy(() => {
+        return {
+          notify: sinon.spy(),
+        };
+      });
+
+      checkForUpdates({
+        name: 'web-ext',
```

Comment:
this call passes a bunch of unnecessary parameters but once you update the Flow type you should get errors about them